src/karma-webpack.js

```diff
@@ -7,9 +7,7 @@
 const os = require('os');
 const path = require('path');
 
-function clone(obj) {
-  return Object.assign({}, obj);
-}
+const clone = require('lodash.clone')
```

Comment:
Better avoid using `lodash.clone` (and `lodash.*`), they are deprecated and don't updated a lot of time, many of them contain security problems

Reply:
This is the only place lodash would be used. 
I am not even sure why we're cloning the webpack config in the first place since it's never been a deep clone, and we're mutating its attributes either way.

Reply:
For info, cloning was added in #48 but as you mentioned, it's a shallow clone, so it in fact does a very poor protection for the original case. I'm thinking to use https://www.npmjs.com/package/clone-deep instead, but this is temporarily blocked until https://github.com/jonschlinkert/shallow-clone/pull/3 is merged and released.

Reply:
Is there any point in cloning the webpack config?

Reply:
We do need to clone the webpack config according to #48.
@jakub-g can you revert to using full lodash then? And use `cloneDeep` instead of `clone`

Reply:
If we're fine using `Object.assign`, can we not just do `Array.isArray(obj) ? obj.map((o) => Object.assign({}, o) : Object.assign({}, obj)` - instead of introducing all of `lodash` as a dependency?

Reply:
I mean from what #48 says, I don't think `Object.assign` is enough (so we should use `lodash` or `deepmerge`), but we should try to avoid `lodash` in it's entirety - I'd pick `deepmerge` personally